src/fsharp/Fsc/Fsc.fsproj

```diff
@@ -30,23 +31,9 @@
       <InProject>false</InProject>
     </FilesToLocalize>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="InternalsVisibleTo.fs" />
-    <Compile Include="..\..\assemblyinfo\assemblyinfo.fsc.exe.fs">
-      <Link>Resources/assemblyinfo.fsc.exe.fs</Link>
-    </Compile>
-    <Compile Include="..\fscmain.fs">
-      <Link>fscmain.fs</Link>
-    </Compile>
-    <None Include="fsc.exe.config">
-      <Link>fsc.exe.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\windows\default.win32manifest">
-      <Link>default.win32manifest</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+
+  <Import Project="Fsc.Sources.proj" />
```

Comment:
Well ... c# has the same issue only much more so ... flavored projects were common 5 or 6 years ago.  Either it gets supported in CPS ore the legacy project systems continue to ship.  For F# it might not be so binary a choice because we never "officially" shipped a flavored template.  However ... I will push back against an attempt to stop those projects working for F# since that was a popular community template for a long while.